File: solutions/0058-length-of-last-word/solution.java
class Solution {
    public int lengthOfLastWord(String s) {
        int i=0;
        for(i=s.length()-1;i>=0 && s.charAt(i)==' ';i--);
        int l=0;
        if(i>=0)
            while(i>=0 && s.charAt(i)!=' ')
            {    l++;i--; }
        
        return l;
    }
}

File: solutions/0058-length-of-last-word/solution.c
int lengthOfLastWord(char* s) {
    int len = 0;
    int i;
    for (i = strlen(s) - 1; i >= 0 && s[i] == ' '; i--);
    for (; i >= 0 && s[i] != ' '; i--) {
        len++;
    }
    
    return len;
}
